src/Microsoft.AspNetCore.Razor.Language/DefaultImportFeature.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal class DefaultImportFeature : RazorProjectEngineFeatureBase, IRazorImportFeature
+    {
+        private IEnumerable<IRazorImportExpander> _expanders;
+
+        public IReadOnlyList<RazorSourceDocument> GetImports(string sourceFilePath)
+        {
+            if (string.IsNullOrEmpty(sourceFilePath))
+            {
+                throw new ArgumentException(Resources.ArgumentCannotBeNullOrEmpty, nameof(sourceFilePath));
+            }
+
+            if (!_expanders.Any())
+            {
+                return Array.Empty<RazorSourceDocument>();
+            }
+
+            var importExpanderContext = new ImportExpanderContext(sourceFilePath);
```

Comment:
I think this is too complicated. We've seen that the requirements for dealing with imports are very simple so far - a *framework* like MVC can provide 100% of the functionality it needs without writing a lot of code.

Reply:
> We've seen that the requirements for dealing with imports are very simple so far - a framework like MVC can provide 100% of the functionality it needs without writing a lot of code.

Can you expand? I view it as being pretty simple.